pkg/kn/traffic/compute.go

```diff
@@ -276,8 +279,17 @@ func Compute(cmd *cobra.Command, targets []servingv1.TrafficTarget, trafficFlags
 	traffic := newServiceTraffic(targets)
 
 	// First precedence: Untag revisions
+	var errTagNames []string
 	for _, tag := range trafficFlags.UntagRevisions {
-		traffic.untagRevision(tag)
+		tagName := traffic.untagRevision(tag, serviceName)
+		if tagName != "" {
+			errTagNames = append(errTagNames, tagName)
+		}
```

Comment:
We're returning the value from function which is passed as its own arguments, 
I think `untagRevsion` could return a bool value based on status of the operation?